pubsub/pubsub-pipe-image/pubsub-to-bigquery.py

```diff
@@ -121,7 +121,9 @@ def write_to_bq(pubsub, sub_name, bigquery):
 
 
 if __name__ == '__main__':
-    sub_name = "tweets-%s" % PROJECT_ID
+    topic_info = PUBSUB_TOPIC.split('/')
+    topic_name = topic_info[-1]
+    sub_name = "tweets%s_%s" % (topic_name, PROJECT_ID)
```

Comment:
PUBSUB_TOPIC looks like this:
projects/<projectname>/topics/<topicname>.
So I think tacking on that full string as part of the subscr name causes issues when it parses the path expression at the other end.  However, if that doesn't sound right I can double check, as when I got the error I'd made a bunch of different changes at the same time.